lib/HttpClient.ts

```diff
@@ -38,6 +38,8 @@ export enum HttpCodes {
 }
 
 const HttpRedirectCodes: number[] = [HttpCodes.MovedPermanently, HttpCodes.ResourceMoved, HttpCodes.SeeOther, HttpCodes.TemporaryRedirect, HttpCodes.PermanentRedirect];
+const HttpRetryCodes: number[] = [HttpCodes.BadGateway, HttpCodes.ServiceUnavailable, HttpCodes.GatewayTimeout];
+const HttpWriteOptions: string[] = ['DELETE', 'POST', 'PUT', 'PATCH'];
```

Comment:
Do we need to expose this?

Reply:
I don't think so - what would it be used for? I'd rather expose less if we don't have a clear use case